samples/templatepack.fsproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageId>FSharp.Data.GraphQL</PackageId>
```

Comment:
It is better to add `.Templates` suffix. See how it is done in Orchard Core and Uno Platform

Reply:
I added the .ProjectTemplates suffix as in Orchard Core. Is it ok?

Reply:
@xperiandri I added the .ProjectTemplates suffix as in Orchard Core. Is it ok?